custom/icds_reports/static/icds_reports/js/spec/location-filter.directive.spec.js

```diff
@@ -36,10 +36,9 @@ describe('Location Filter Controller', function () {
     ];
 
     beforeEach(module('icdsApp', function ($provide) {
-        $provide.constant("userLocationId", null);
+        utils.provideDefaultConstants($provide, {});
         $provide.constant("allUserLocationId", []);
         $provide.constant("haveAccessToAllLocations", true);
```

Comment:
can you not remove more from here?

Reply:
haveAccessToAllLocations can be removed. I'll remove it